### ISSUE TITLE
events not always emitted from `Identity` instance in some browsers

### ISSUE THREAD

**Issue by Kicu** [schibsted/account-sdk-browser#40]
Hello

**tl;dr:** If you go to https://pro.sdk-example.com/ (or the PRE version for that matter) and log in, the `login` event won't be emitted every time on **chrome** or **firefox**.

Looking at this: https://github.com/schibsted/sdk-example/blob/master/browser/index.js#L29
I expect (and I think it's the correct assumption) that every time "Update" button is clicked (which means `hasSession` is called), then proper console.log message should appear.
This behaviour currently only works in Safari, and not on chrome or firefox. On both of these browsers nothing happens as if the event is not emitted.

What I also observed happens is that on safari there is an actual call made to `session.login.schibsted.com` every time the button is clicked. That is not the case on chrome/ff - no network activity there. Seems to me that the events are not triggered because there is no actual calls made from browser.
Session info still indicates that a user is logged in and the result of `hasSession` has the proper info. It's just the events that are missing.

I will be very thankful for a follow-up on this, seems to be a pretty crucial issue and we used to rely on `login` and `notLoggedIn` events a lot on 2x version.

`chrome version 66`
`firefox version 60`

Thanks
